console-subscriber/src/builder.rs

```diff
@@ -145,6 +149,16 @@ impl Builder {
         }
     }
 
+    /// Sets the filter environment variable for tracing events.
+    ///
+    /// By default, this is `RUST_LOG`.
+    pub fn filter_env_variable(self, filter_env_variable: impl Into<String>) -> Self {
```

Comment:
I think maybe we should rename this to `filter_env_var` rather than `filter_env_variable`? It feels a little bit weird to me that we abbreviate "environment" to "env" but we don't abbreviate "variable" to "var", especially since the stdlib function is `std::env::var` rather than `std::env::variable`.

i know i was the one who suggested this name originally, sorry about that.